plugins/woocommerce/includes/abstracts/abstract-wc-payment-gateway.php

```diff
@@ -204,8 +205,17 @@ public function get_method_description() {
 	 * Output the gateway settings screen.
 	 */
 	public function admin_options() {
+		$is_reactify_enabled      = Features::is_enabled( 'reactify-classic-payments-settings' );
+		$offline_payment_gateways = array( 'bacs', 'cheque', 'cod' );
+		$is_offline_gateway       = in_array( $this->id, $offline_payment_gateways, true );
+
+		$return_url = admin_url( 'admin.php?page=wc-settings&tab=checkout' );
+		if ( $is_reactify_enabled && $is_offline_gateway ) {
+			$return_url = add_query_arg( 'section', 'offline', $return_url );
+		}
+
 		echo '<h2>' . esc_html( $this->get_method_title() );
-		wc_back_link( __( 'Return to payments', 'woocommerce' ), admin_url( 'admin.php?page=wc-settings&tab=checkout' ) );
+		wc_back_link( __( 'Return to payments', 'woocommerce' ), $return_url );
```

Comment:
Question to @orcunomattic:

Since the latest Figma designs don't have a screen for a single offline PM (eg. direct bank transfer), are we okay with the current title + backlink design (see the screenshot in PR description)? The back button doesn't match the iOS-like button you have used in Figma.